site/docs/spark-quickstart.md

```diff
@@ -100,7 +100,7 @@ services:
       - AWS_ACCESS_KEY_ID=admin
       - AWS_SECRET_ACCESS_KEY=password
       - AWS_REGION=us-east-1
-    entrypoint: >
+    entrypoint: |
```

Comment:
Indeed, `>` collapses the code into one line, but do we need one or multi-line here? IMHO it's one line because it's one command (`/bin/sh -c "..."`)